src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+  "item.mythicmetals.prometheum_axe": "Prometheumaxt",
+  "item.mythicmetals.prometheum_boots": "Prometheumstiefel",
+  "item.mythicmetals.prometheum_chestplate": "Prometheumbrustplatte",
+  "item.mythicmetals.prometheum_dust": "Prometheumstaub",
+  "item.mythicmetals.prometheum_helmet": "Prometheumhelm",
+  "item.mythicmetals.prometheum_hoe": "Prometheumhacke",
+  "item.mythicmetals.prometheum_ingot": "Prometheumbarren",
+  "item.mythicmetals.prometheum_leggings": "Prometheumhose",
+  "item.mythicmetals.prometheum_nugget": "Prometheumnugget",
+  "item.mythicmetals.prometheum_pickaxe": "Prometheumspitzhacke",
+  "item.mythicmetals.prometheum_shovel": "Prometheumschaufel",
+  "item.mythicmetals.prometheum_sword": "Prometheumschwert",
+  "item.mythicmetals.quadrillum_anvil": "Quadrillumamboss",
+  "item.mythicmetals.quadrillum_axe": "Quadrillumaxt",
+  "item.mythicmetals.quadrillum_dust": "Quadrillumstaub",
+  "item.mythicmetals.quadrillum_hoe": "Quadrillumhacke",
+  "item.mythicmetals.quadrillum_ingot": "Quadrillumbarren",
+  "item.mythicmetals.quadrillum_nugget": "Quadrillumnugget",
+  "item.mythicmetals.quadrillum_pickaxe": "Quadrillumspitzhacke",
+  "item.mythicmetals.quadrillum_shovel": "Quadrillumschaufel",
+  "item.mythicmetals.quadrillum_sword": "Quadrillumschwert",
+  "item.mythicmetals.raw_adamantite": "Rohes Adamantit",
+  "item.mythicmetals.raw_aquarium": "Rohes Aquarium",
+  "item.mythicmetals.raw_banglum": "Rohes Banglum",
+  "item.mythicmetals.raw_carmot": "Rohes Carmot",
+  "item.mythicmetals.raw_kyber": "Rohes Kyber",
+  "item.mythicmetals.raw_manganese": "Rohes Manganese",
+  "item.mythicmetals.raw_midas_gold": "Rohes Midasgold",
+  "item.mythicmetals.raw_mythril": "Rohes Mythril",
+  "item.mythicmetals.raw_orichalcum": "Rohes Orichalcum",
+  "item.mythicmetals.raw_osmium": "Rohes Osmium",
+  "item.mythicmetals.raw_palladium": "Rohes Palladium",
+  "item.mythicmetals.raw_platinum": "Rohes Platin",
+  "item.mythicmetals.raw_prometheum": "Rohes Prometheum",
+  "item.mythicmetals.raw_quadrillum": "Rohes Quadrillum",
+  "item.mythicmetals.raw_runite": "Rohes Runite",
+  "item.mythicmetals.raw_silver": "Rohes Silber",
+  "item.mythicmetals.raw_stormyx": "Rohes Stormyx",
+  "item.mythicmetals.raw_tin": "Rohes Zinn",
+  "item.mythicmetals.red_aegis_sword": "Schneide des Aegis",
+  "item.mythicmetals.runite_anvil": "Runiteamboss",
+  "item.mythicmetals.runite_axe": "Runiteaxt",
+  "item.mythicmetals.runite_arrow": "Runitepfeil",
+  "item.mythicmetals.tipped_runite_arrow": "Tipped Runitepfeil",
+  "item.mythicmetals.tipped_runite_arrow.effect.empty": "Uncraftable Tipped Runitepfeil",
+  "item.mythicmetals.tipped_runite_arrow.effect.water": "Runitepfeil des Splishy Splashy",
+  "item.mythicmetals.tipped_runite_arrow.effect.mundane": "Mundane Runitepfeil",
+  "item.mythicmetals.tipped_runite_arrow.effect.thick": "Minimal dickerer Runitepfeil",
+  "item.mythicmetals.tipped_runite_arrow.effect.awkward": "Weird Runitepfeil",
```

Comment:
Are these accurate compared to vanilla translations? They seem untranslated

Reply:
Yep, that was untranslated. Should be accurate to vanilla now.